packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -430,6 +430,8 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
   const callees = useSelector(getTargetCallees) as StartCallIdentifier[];
   const locale = useLocale();
   const palette = useTheme().palette;
+  /* @conditional-compile-remove(PSTN-calls) */
+  const alternateCallerId = adapter.getState().alternateCallerId;
```

Comment:
Is this never changed or it could be potentially changed after it is rendered?
If that is a potential change, maybe useSelector would be a good option (which will trigger re-render when something changed in state)